XamlStyler.Core/DocumentProcessors/ProcessInstructionDocumentProcessor.cs

```diff
@@ -29,7 +29,13 @@ public void Process(XmlReader xmlReader, StringBuilder output, ElementProcessCon
                 output.Append(Environment.NewLine);
             }
 
-            output.Append(currentIndentString).Append("<?Mapping ").Append(xmlReader.Value).Append(" ?>");
+            output
+                .Append(currentIndentString)
+                .Append("<?")
+                .Append(xmlReader.Name)
+                .Append(" ")
+                .Append(xmlReader.Value)
+                .Append(" ?>");
```

Comment:
When we are in here, can we update this line to use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated): ```$"{currentIndentString}<?{xmlReader.Name} {xmlReader.Value}?>"```? Note removal of that last space--it appears that [XML Processing Instructions](https://en.wikipedia.org/wiki/Processing_Instruction) don't require it. This will require updating the test too.

Reply:
Sure.

I just wanted to stay consistent with the previously used style - I do prefer the interpolation syntax over the `StringBuilder` calls.

It will be changed with the next commits.

Reply:
I updated it with https://github.com/Xavalon/XamlStyler/pull/215/commits/36cdae1f3e60fb1d6a56793168b893831a6bc635

Looking into the test files I am a bit irritated by the last whitespace inside the `<xml... ?>` node. 
Looks a bit inconsistent to me, but I am no XML-guru at all.

Maybe I would open an additional Issue/PR for the XML declaration tag update for you to consider ;-)